po4a

```diff
@@ -1492,6 +1498,17 @@ if (not $po4a_opts{"no-translations"}) {
             $doc->{TT}{po_in} = $po;
             $doc->{TT}{po_in}->stats_clear();
 
+	    my $translationBackupfile;
```

Comment:
The name of this variable is very different from the other ones. Not a big deal but if you have a better idea, it'd be welcome.

Reply:
simply translation_backup?

Reply:
great!